e2e/bitrise.yml

```diff
@@ -0,0 +1,118 @@
+format_version: "11"
+default_step_lib_source: https://github.com/bitrise-io/bitrise-steplib.git
+
+app:
+  envs:
+  # Shared secrets for testing, use a .bitrise.secrets.yml file to define these locally
+  - BITRISEIO_CACHE_SERVICE_URL: $BITRISEIO_CACHE_SERVICE_URL
+  - CACHE_API_CLIENT_SECRET: $CACHE_API_CLIENT_SECRET
+
+workflows:
+  test_spm:
+    envs:
+    - TEST_APP_URL: https://github.com/bitrise-io/sample-apps-ios-swiftpm.git
+    - BRANCH: master
+    before_run:
+    - _generate_api_token
+    - _setup
+    steps:
+    - change-workdir:
+        title: Switch working dir to _tmp/sample-swiftpm4
+        inputs:
+        - path: ./_tmp/sample-swiftpm4
+    - script:
+        title: Clean DerivedData
+        inputs:
+        - content: rm -rf ~/Library/Developer/Xcode/DerivedData/
+    - script:
+        title: Resolve dependencies
+        inputs:
+        - content: |-
+            set -ex
+            xcodebuild -resolvePackageDependencies
+    - path::./:
+        title: Execute step
+        run_if: "true"
+        is_skippable: false
+        inputs:
+        - verbose: "true"
+    - script:
+        title: Clean DerivedData
+        inputs:
+        - content: |-
+            rm -rf ~/Library/Developer/Xcode/DerivedData/
+    - restore-cache:
+        run_if: "true"
+        is_skippable: false
+        inputs:
+        - key: "{{ .OS }}-{{ .Arch }}-spm-cache-debug-"
+        - verbose: "true"
+    - script:
+        title: Check if restored folder is empty
```

Comment:
I'm not sure if I get this :) What is this for? This doesn't seem to do anything.